libbpfgo/helpers/rwArray.go

```diff
@@ -0,0 +1,76 @@
+package helpers
+
+import (
+	"sync"
+)
+
+// Like RWMutex, RWArray allows for multiple concurrent readers but
```

Comment:
Can you format this comment to be in GoDoc format (i.e. directly above the RWArray definition, and starting with "RWArray ..."